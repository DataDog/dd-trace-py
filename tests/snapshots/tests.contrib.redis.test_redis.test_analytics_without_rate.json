[[
  {
    "name": "redis.command",
    "service": "redis",
    "resource": "GET cheese",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "redis",
    "error": 0,
    "meta": {
      "_dd.p.dm": "-0",
      "component": "redis",
      "db.system": "redis",
      "language": "python",
      "out.host": "localhost",
      "redis.raw_command": "GET cheese",
      "runtime-id": "b8f85bec81c04f81aa7a1e60a8dd199f",
      "span.kind": "client"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_dd1.sr.eausr": 1.0,
      "_sampling_priority_v1": 1,
      "network.destination.port": 6379,
      "out.redis_db": 0,
      "process_id": 773,
      "redis.args_length": 2
    },
    "duration": 2804000,
    "start": 1667243545616006000
  }]]
