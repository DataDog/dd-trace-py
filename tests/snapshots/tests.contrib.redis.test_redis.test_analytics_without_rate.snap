[[{"name" "redis.command"
   "service" "redis"
   "resource" "GET cheese"
   "type" "redis"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1600443665674164000
   "duration" 2844000
   "meta" {"runtime-id" "81d7379fb04f420c9b55d9ee93ca4ea4"
           "redis.raw_command" "GET cheese"
           "out.host" "localhost"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_dd1.sr.eausr" 1.0
              "_dd.tracer_kr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 48090
              "out.port" 6379
              "out.redis_db" 0
              "redis.args_length" 2}}]]
